src/overlays/actors/ovl_Obj_Tree/z_obj_tree.c

```diff
@@ -125,7 +125,7 @@ void ObjTree_Destroy(Actor* thisx, GlobalContext* globalCtx) {
     ObjTree* this = THIS;
     s32 bgId;
 
-    if (!(this->dyna.actor.params & 0x8000)) {
+    if (!(OBJTREE_ISLARGE(&this->dyna.actor))) {
```

Comment:
```suggestion
    if (!OBJTREE_ISLARGE(&this->dyna.actor)) {
```
Parenthesis shouldn't be necessary.